app/components/pageLayout/pageLayout.tsx

```diff
@@ -14,10 +14,11 @@ import { ADOBE_EMBED_URL } from "app/config/constants";
 import Script from "next/script";
 import { trackVirtualPageView } from "app/utils/utils";
 import { usePathname } from "next/navigation";
+import { BreadcrumbsDataProps } from "@nypl/design-system-react-components/dist/src/components/Breadcrumbs/Breadcrumbs";
```

Comment:
use the existing import statement.

Reply:
Is there one? I did not even notice this import wasn't the standard one, but it looks like we don't export those props from the DS for some reason

Reply:
Ugh, I see why you did that. I'll work on that in the DS. So this is ok.